tests/checkers/rst-yamllint.py

```diff
@@ -0,0 +1,317 @@
+"""Sanity test using rstcheck and sphinx."""
+from __future__ import annotations
+
+import io
+import os
+import re
+import sys
+import tempfile
+import traceback
+
+from yamllint import linter
+from yamllint.config import YamlLintConfig
+# from yamllint.linter import PROBLEM_LEVELS
+
+from docutils import nodes
+from docutils.core import Publisher
+from docutils.frontend import Values
+from docutils.io import StringInput
+from docutils.parsers.rst import Directive, Parser as RstParser
+from docutils.parsers.rst.directives import register_directive, unchanged as directive_param_unchanged
+from docutils.utils import new_reporter, Reporter, SystemMessage
+
+import sphinx.application
+from sphinx.application import Sphinx
+
+
+YAMLLINT_CONFIG = r"""
```

Comment:
Why not have this in a conventional `.yamllint` file? Seems odd to add a linter for YAML, but then stick an unlintable piece of YAML back into a Python module.

Reply:
3d6aaff4537283facda4348271966618395dacfc